apps/filebrowser/src/filebrowser/api.py

```diff
@@ -211,6 +211,25 @@ def rmtree(request):
   return HttpResponse(status=200)
 
 
+@error_handler
+def get_trash_path(request):
+  path = _normalize_path(request.GET.get('path'))
+  response = {}
+
+  trash_path = request.fs.trash_path(path)
```

Comment:
Nit: reuse this in the join below?

Reply:
We are using the `trash_path` in the join below:
```python
user_home_trash_path = request.fs.join(request.fs.current_trash_path(trash_path), request.user.get_home_directory().lstrip('/'))
```

`request.fs.current_trash_path(trash_path)` helps in creating the intermediary path.

Reply:
True, I misread.